libkbfs/file_data.go

```diff
@@ -509,44 +535,298 @@ func (fd *fileData) createIndirectBlock(
 	return fblock, nil
 }
 
-// newRightBlock appends a new block (with a temporary ID) to the end
-// of this file.
+// newRightBlock creates space for a new rightmost block, creating
```

Comment:
The old comment did say "newRightBlock appends a new block (with a temporary ID) to the end of this file."